packages/nimble-components/src/rich-text-editor/index.ts

```diff
@@ -70,6 +70,10 @@ export class RichTextEditor extends FoundationElement {
     private readonly markdownSerializer = this.initializeMarkdownSerializer();
     private readonly domSerializer = DOMSerializer.fromSchema(schema);
     private readonly xmlSerializer = new XMLSerializer();
+    private readonly tipTapNodeNames = {
+        bulletList: 'bulletList',
+        numberedList: 'orderedList'
+    } as const;
```

Comment:
Follow the nimble enum coding conventions given here - [constant enum](https://github.com/ni/nimble/blob/32c8dd6d1e921581b13eb4ac82d5f0331d323752/packages/nimble-components/docs/coding-conventions.md#use-const-objects-instead-of-typescript-enums)

Reply:
An example in nimble banner component [here](https://github.com/ni/nimble/blob/8c748c8f5e1b67bb7f2c984f1a91c14f967108c8/packages/nimble-components/src/banner/types.ts)